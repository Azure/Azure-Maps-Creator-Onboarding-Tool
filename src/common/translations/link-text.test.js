import { render } from '@testing-library/react';

import LinkText from './link-text';

describe('LinkText', () => {
  it('should render component', () => {
    const view = render(<LinkText href='https://www.pure-truth.com/'>I am a nobody. Nobody is perfect. Therefore, I am perfect.</LinkText>);
    expect(view.container).toMatchSnapshot();
  });
});