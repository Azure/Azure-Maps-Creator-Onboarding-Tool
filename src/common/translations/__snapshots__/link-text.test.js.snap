// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkText should render component 1`] = `
<div>
  <a
    href="https://www.pure-truth.com/"
    rel="noreferrer"
    target="_blank"
  >
    I am a nobody. Nobody is perfect. Therefore, I am perfect.
  </a>
</div>
`;
